homeassistant/components/media_player/russound_rnet.py

```diff
@@ -76,13 +75,16 @@ class RussoundRNETDevice(MediaPlayerDevice):
     """Representation of a Russound RNET device."""
 
     def __init__(self, hass, russ, sources, zone_id, extra):
-        """Initialise the Russound RNET device."""
+        """Initialise the Russound RNET device.
+
+        Note: now uses the Russound device directly to obtain the state and
+        volume level, and therefore we no longer
+        have properties in this class for these to variables.
```

Comment:
This comment block can be removed. It's good to note in the PR, but after it gets merged, the comment won't make sense any more.